src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -1216,6 +1227,15 @@ public void onWorldTick(@NotNull final TickEvent.WorldTickEvent event)
                     spawnCitizen();
                 }
             }
+
+            if (event.world.getDifficulty() != EnumDifficulty.PEACEFUL && Configurations.doBarbariansSpawn)
+            {
+                if (MobEventsUtils.isItTimeToRaid(event.world, this))
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Merge this if statement with the enclosing one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1066)